### DISCUSSION TITLE
Org Owners

### DISCUSSION THREAD

**Discussion by mrosatii** [semaphoreci/semaphore#250] [Documentation improvements]
https://docs.semaphoreci.com/using-semaphore/rbac#org-owner

- **Documentation states**: They (Owners) cannot be removed from the organization.

- **Documentation also states**: If you need to demote or remove an owner from the organization, any user with the Owner permission level needs to send an email to [support@semaphoreci.com](mailto:support@semaphoreci.com) from the primary email address associated with their GitHub or Bitbucket account used to log into Semaphore. (https://docs.semaphoreci.com/using-semaphore/organizations#remove-owner)

Since organization owners can be removed, we could consider updating the docs by removing the sentence: 'They cannot be removed from the organization' in [this section](https://docs.semaphoreci.com/using-semaphore/rbac#org-owner)

**Comment by ivanaur** [comment]
@mrosatii Good catch! 🙌 Could you create a pull request for this change and assign it to @TomFern and me?

**Reply by mrosatii** [reply]
Will do 🙌🏻

**Reply by mrosatii** [reply]
Hi there @ivanaur @TomFern 👋🏼 I created this [PR](https://github.com/mrosatii/semaphorecont/pull/1) for updating the docs but I'm not sure why I can't assign it to both of you 🤔. I sent invites to both of you to join my repo as collaborators; maybe once you accept the system will let me assign the PR to you?

**Comment by TomFern** [comment]
Hi @mrosatii. You should be able to create a PR across repos, ie. from your forked repo to the main repo semaphoreci/semaphore repo.

To do that:
1. In your repo, create a new PR
2. In base repository (left) select "semaphoreci/semaphore" branch: main
3. In head repository (right) select your repo, your branch
4. Press Create Pull request

![CleanShot 2024-11-10 at 12 01 33@2x](https://github.com/user-attachments/assets/21ca39bc-5780-46ba-b1f3-997431ef57f1)

This should create the PR on the "semaphoreci/semaphore" instead of on your own. There we can review it and merge in into the main branch.

**Comment by mrosatii** [comment]
Thanks for your help @TomFern 🙌🏻 I just created the [PR](https://github.com/semaphoreci/semaphore/pull/275). Is there a way I can assign it to Ivana as well? I see is assigned to you and Aleksandar but It seems I can't add more people

**Reply by TomFern** [reply]
I think we can only assign people with collaborator access in this repo. That's why you can't assign her. 

Would you like us to wait for her to review the changes before we merge them?

**Reply by mrosatii** [reply]
@TomFern yes please, that would be good. Thanks! 🙌🏻

**Reply by ivanaur** [reply]
Thank you both, the changes are good 👍 🍫

**Comment by TomFern** [comment]
Thank you @ivanaur. I pushed the changes into prod
